safe/gis/vector/multi_buffering.py

```diff
@@ -122,8 +123,9 @@ def multi_buffering(layer, radii, callback=None):
             callback(current=i, maximum=feature_count, step=processing_step)
 
     # We transfer keywords to the output.
-    buffered.keywords = layer.keywords
+    buffered.keywords = KeywordIO().read_keywords(layer)
```

Comment:
I found that 'layer.keywords' give nothing, it didn't return any keywords. So I use keyword io. Is it fine?